spec/features/admin/settings_spec.rb

```diff
@@ -89,4 +89,48 @@
 
   end
 
-end
+  describe "Skip verification" do
+
+    scenario "deactivate verification", :js do
+      Setting["feature.user.skip_verification"] = 'true'
+      setting = Setting.where(key: "feature.user.skip_verification").first
+      admin = create(:administrator).user
+      login_as(admin)
+
+      visit admin_settings_path
+
+      accept_alert do
+        find("#edit_setting_#{setting.id} .button").click
+      end
+
+      logout
+
+      user = create(:user)
+      login_through_form_as(user)
+
+      expect(page).to have_current_path(welcome_path)
+    end
+
+    scenario "activate verification", :js do
+      Setting["feature.user.skip_verification"] = nil
+      setting = Setting.where(key: "feature.user.skip_verification").first
+      admin = create(:administrator).user
+      login_as(admin)
+
+      visit admin_settings_path
+
+      accept_alert do
+        find("#edit_setting_#{setting.id} .button").click
+      end
+
+      logout
+
+      user = create(:user, sign_in_count: 1)
```

Comment:
Shouldn't both scenarios be the same? We're trying to see if the having the setting enabled/disabled makes a difference... but you're changing the conditions on both so ...what are we really checking here?